src/autolabel/tasks/attribute_extraction.py

```diff
@@ -268,14 +273,17 @@ def parse_llm_response(
 
         if successfully_labeled:
             for attribute in self.config.attributes():
-                attr_options = attribute.get("options")
+                attr_options, attr_type = attribute.get("options"), attribute.get(
+                    "task_type"
+                )
                 if attr_options is not None and len(attr_options) > 0:
                     attr_label = str(llm_label.get(attribute["name"]))
-                    if attr_label is not None and attr_label not in attr_options:
-                        logger.warning(
-                            f"Attribute {attr_label} from the LLM response {llm_label} is not in the labels list"
-                        )
-                        llm_label.pop(attribute["name"], None)
+                    if attr_type == TaskType.CLASSIFICATION:
```

Comment:
also check that every label that we get for multilabel is one of the options if the guidelines were followed

Reply:
We shouldn't need this either moving forward with guided generation. Will be removing classification as well soon!